samples/resource-server-sample/src/main/resources/application.yaml

```diff
@@ -0,0 +1,8 @@
+# See https://docs.spring.io/spring-security/site/docs/current/reference/html5/#oauth2resourceserver 
+# for more configuration options
+spring:
+  security:
+    oauth2:
+      resourceserver:
+        jwt:
+          issuer-uri:  https://accounts.google.com
```

Comment:
Let's use the `jwk-set-uri` property instead, that way the resource server will at least startup.

We could also mock the jwkSet endpoint using `MockWebServer`. Take a look at `OAuth2ResourceServerConfigurerTests` in Spring Security for an example on how to set it up.

Reply:
With this one, I will need some more time as I have not used the `MockWebServer` before, I'll do some research, check the example and try to finish this later today.

Reply:
Ok, I checked out the sample from OAuth2ResourceServerConfigurerTests with the MockWebServer, but It started to feel like a lot's work to get the token validation mocked that way. Instead, I decided to mock the JwtDecoder. Let me know what you think.